Cargo.toml

```diff
@@ -22,6 +22,11 @@ log = "0.4.6"
 users = "0.11.0"
 page_size = "0.4.2"
 serde = {version = "1.0.102", features = ["std", "derive"], optional = true}
+pool = "0.1.4"
+tokio = {version = "1.1.0", features = ["full"]}
+async-trait = "0.1.42"
+futures = "0.3.12"
+dashmap = "4.0.2"
```

Comment:
only used in the XMP example, probably need to just pull that out into its own crate as an example now to avoid these deps.

Reply:
:+1: